docs/src/usage/analysis.md

```diff
@@ -200,6 +200,41 @@ save("tsa.png", tsa_fig)
 ![Plots of Temporal Sensitivities](/ADRIA.jl/dev/assets/imgs/tsa.png?raw=true "Temporal Sensitivity Analysis")
 
 
+### Convergence Analysis for Sensitivity
+
+When undertaking sensitivity analysis it is importnant to have a sufficient sample size that the sensitivity measure has converged and is relatively stable. To understand convergence behaviour a 
+large sample can be drawn and split into smaller sub-samples. Plotting a sensitivity measure against increasing sample size can help to understand whether the sample size is sufficient for convergence.
+The function `convergence` can be used to calculated a sensitivity measure for an increasing number of samples. The result can then be plotted as band plots or a heat map using `viz.convergence`.
```

Comment:
You say `convergence` and later say `viz.convergence` to differentiate between the analysis and visualization methods.

Should the first be `sensitivity.convergence` or `analysis.convergence` instead?